src/index.ts

```diff
@@ -99,7 +106,18 @@ program
         "Toggle whether or not the block is public."
     )
     .action(({ togglePublic }: any) => {
-        publishUpdate(togglePublic);
+        update(togglePublic);
+    });
+
+program
+    .command("release")
+    .description(
+        `Release your existing block in the Block Theme Registry
```

Comment:
What's the meaning of release. (We should clarify that it will become public, etc...)

Reply:
It becomes "public" in the sense that people can see it in the public store, but release doesn't mean it can be used by other stores.